nuxeo-ai-core/src/main/java/org/nuxeo/ai/enrichment/AbstractEnrichmentService.java

```diff
@@ -101,4 +113,19 @@ public String saveJsonAsRawBlob(String rawJson) {
     protected List<TagSuggestion> asTags(List<AIMetadata.Tag> tags) {
         return Collections.singletonList(new TagSuggestion(suggestionProperty, tags));
     }
+
+    protected KeyValueStore getStore() {
```

Comment:
This class has an existing method to get the `transientStoreName`, and its configurable in the `init` method.  You  probably need to differentiate between that store and your `getStore` method.  At least some javadoc.

Reply:
it will return the dedicated one now